src/chain/ethereum.rs

```diff
@@ -145,7 +145,9 @@ impl PevmChain for PevmEthereum {
             access_list: tx.access_list.unwrap_or_default().into(),
             blob_hashes: tx.blob_versioned_hashes.unwrap_or_default(),
             max_fee_per_blob_gas: tx.max_fee_per_blob_gas.map(U256::from),
-            authorization_list: None, // TODO: Support in the upcoming hardfork
+            authorization_list: Some(AuthorizationList::Signed(
```

Comment:
Intuitively we'd want a `None` here when `tx.authorization_list` is `None` instead of defaulting to a `Some`. This may also be why the test is failing. Let's just `map` it?